cypress/integration/listingFlows/viewListing.spec.js

```diff
@@ -42,4 +42,22 @@ describe('View listing', () => {
     cy.get('@listingsModal').findByRole('button', { name: /Close/ }).click();
     cy.get('@listingTitle').should('have.focus');
   });
+
+  it('redirects when a user contacts via connect', () => {
```

Comment:
Documentation (blocking): Make it clearer in the test description that this is for a logged out  member trying to respond to a listing that gets redirected to the login. I don't know what the best wording is for this. Another option could be to put this it in a `describe` for a logged out member.